packages/react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -91,7 +101,12 @@ export const MessageBarBase: React.FunctionComponent<IMessageBarProps> = React.f
             <Icon iconName={ICON_MAP[messageBarType!]} className={classNames.icon} />
           )}
         </div>
-        <div className={classNames.text} id={labelId} role="status" aria-live={getAnnouncementPriority(messageBarType)}>
+        <div
+          className={classNames.text}
+          id={labelId}
+          role={getRole(messageBarType)}
```

Comment:
There's already a `IMessageBarProps.role` due to the native props inheritance, and we should probably apply that here instead if it's provided. Be sure to also add `role` to the list of props to ignore in the `getNativeProps` call on line 64 so it's not also applied to the inner div.

Also for clarity, it would be good to explicitly add `role?: string` to `IMessageBarProps`, with a note of the default.

Reply:
Sounds good. I've made those updates: https://github.com/microsoft/fluentui/pull/18042/commits/2199899f169bea261217f1c046a728b8210967e9, https://github.com/microsoft/fluentui/pull/18042/commits/2a87275579dfbccf6925bac84d2a075664f0f813.

For the type on `role`, I have it as `string` right now, but would it make sense to use `AriaRole` from [accessibility/src/types.ts#L82-L87](https://github.com/microsoft/fluentui/blob/master/packages/fluentui/accessibility/src/types.ts#L82-L87) instead? (It doesn't look like `@fluentui/react` currently uses that dependency though)

Reply:
Thanks for the updates! Having a proper type would be nice, but that package is only intended for use with `@fluentui/react-northstar`.